lib/get-xo.js

```diff
@@ -0,0 +1,19 @@
+/** @babel */
+import {allowUnsafeNewFunction} from 'loophole';
+import getPackagePath from './get-package-path';
+
+// (base: string) => xo: Promise<Object>
+export default function getXO(base) {
+	const getResolvePath = () => base ? getPackagePath(base) : Promise.resolve();
+
+	return getResolvePath()
```

Comment:
We can leave it async for now and do some benchmarking when all the pending changes are done. Mind opening an issue about benchmarking this so we don't forget?